contrib/utxo-tools/utxo_to_sqlite.py

```diff
@@ -154,13 +115,13 @@ def main():
         # read key (COutPoint)
         if coins_per_hash_left == 0:  # read next prevout hash
             prevout_hash = f.read(32)[::-1].hex()
-            coins_per_hash_left = read_compactsize(f)
-        prevout_index = read_compactsize(f)
+            coins_per_hash_left = deser_compact_size(f)
```

Comment:
Seems to me `deser_compact_size` does and extra little endian conversion - for a single byte, which seems redundant, we might as well remove it from there to match this implementation